src/main/java/com/epam/javacourse/hometask1/HelloWorld.java

```diff
@@ -0,0 +1,7 @@
+package com.epam.javacourse.hometask1;
+
+public class HelloWorld {
+  public static void main(String[] args) {
```

Comment:
As per Java code convention you should use 4 spaces for indentation

Reply:
Sorry, these settings are used for another project and applied to this one by mistake. Corrected.